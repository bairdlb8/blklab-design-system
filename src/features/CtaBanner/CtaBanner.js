import React from 'react'
import PropTypes from 'prop-types'

import './CtaBanner.scss'

const CtaBanner = ({className=''}) => {
  return <div>CtaBanner</div>
}

CtaBanner.propTypes = {}

export default CtaBanner
