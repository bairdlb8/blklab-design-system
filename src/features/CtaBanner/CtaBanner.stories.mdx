import { Meta, Story, Props, Preview } from '@storybook/addon-docs/blocks'

import CtaBanner from './CtaBanner'

<Meta title="Features / CtaBanner" component={ CtaBanner } />

# CtaBanner

## Add some text here

export const Template = (args) => {
  return <CtaBanner {...args} />
}

<Preview withToolbar>
  <Story name="Primary" args={ {} }>
    {Template.bind({})}
  </Story>
</Preview>

<Props of={ CtaBanner } />
