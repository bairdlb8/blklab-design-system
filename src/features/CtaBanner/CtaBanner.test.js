import React from 'react'
import { render } from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'
import { toHaveNoViolations } from 'jest-axe'
import CtaBanner from './CtaBanner'

expect.extend(toHaveNoViolations)

describe('<CtaBanner />', () => {

  describe('accessibility', () => {
    it('should not fail any accessibility tests', async () => {
      const { container } = render(<CtaBanner />)
      expect(await axe(container)).toHaveNoViolations()
    })
  })

  it('should take a snapshot', () => {
    const { asFragment } = render(<CtaBanner />)

    expect(asFragment(<CtaBanner />)).toMatchSnapshot()
  })

})
