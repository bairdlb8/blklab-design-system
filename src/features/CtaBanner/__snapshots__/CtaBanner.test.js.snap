// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CtaBanner /> should take a snapshot 1`] = `
<DocumentFragment>
  <div>
    CtaBanner
  </div>
</DocumentFragment>
`;
